api/tests/test_utils/db_utils.py

```diff
@@ -204,13 +204,13 @@ def empty_database(db, url):
         )
 
         try:
-            for table_name in tables_to_delete:
-                table = Base.metadata.tables[table_name]
-                delete_stmt = delete(table)
-                db.session.execute(delete_stmt)
+            with db.start_db_session() as session:
+                for table_name in tables_to_delete:
+                    table = Base.metadata.tables[table_name]
+                    delete_stmt = delete(table)
+                    session.execute(delete_stmt)
 
-            db.commit()
+                session.commit()
```

Comment:
Do we need the commit here? I thought the commit would be done in start_db_session. I guess an extra commit is not a big deal.

Reply:
removed.